about.html

```diff
@@ -12,52 +12,91 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+  <nav>
+    <img src="img/lambda-black.png" alt="lambda logo">
+    <a href="index.html">Home</a>
+    <a href="about.html">About</a>
+    <a href="#products">Products</a>
+    <a href="#blog">Blog</a>
+    <a href="#contact">Contact</a>
+</nav>
```

Comment:
Love the semantic "nav " you placed in line 16 and 23, a way to make this more semantic is by implementing the <header> element tag that is used to introduce navigation links or introductory content. Great job.